doc/doxygen/cantera.bib

```diff
@@ -214,6 +279,16 @@ @article{niemeyer2017
     url = {https://dx.doi.org/10.1016/j.cpc.2017.02.004},
     volume = {21},
     year = {2017}}
+@article{pedersen1993,
+    author = {T.~Pedersenand R.~.C.~Brown},
```

Comment:
```suggestion
@article{pederson1993,
    author = {T.~Pederson and R.~.C.~Brown},
```
Elsewhere it's "Pederson"? ...

Reply:
Ah, thanks. It's definitely "Pedersen", if you follow the DOI link. I corrected the incorrect spelling in `IonFlow.h`.